test/utils/createDOM.js

```diff
@@ -23,7 +23,6 @@ const blacklist = ['sessionStorage', 'localStorage'];
 function createDOM() {
   const dom = new JSDOM('', {
     pretendToBeVisual: true,
-    url: 'http://localhost', // https://github.com/jsdom/jsdom/issues/2383
   });
```

Comment:
I think that this could also be a good alternative.

```suggestion
    url: 'http://localhost',
  });
```